extensions/websockets-next/server/runtime/src/main/java/io/quarkus/websockets/next/WebSocketConnection.java

```diff
@@ -29,6 +29,12 @@ public interface WebSocketConnection extends Sender, BlockingSender {
      */
     String id();
 
+    /**
+     *
+     * @return the fully qualified class name of the {@link WebSocket} endpoint
+     */
+    String endpoint();
```

Comment:
This is more like an _endpoint id_, I can change the javadoc. I'd like to avoid using `Class<?>` here.

Reply:
Well, the method name does not contain the `class` at all :wink:.

But yes, we could improve this. The main goal of this "property" is to allow users to filter connections of a specific endpoint. `Class<?>` seems to be an obvious solution but I'd like to try to avoid loading the class here, hence the string.

Reply:
Hm, that's actually a nice idea. I'd call it endpoint id. And it could be defaulted to FQCN.

Reply:
Exactly what I had in mind :wink:

Reply:
Done.

Reply:
Nice! Thanks!